### DISCUSSION TITLE
请问谷歌云盘 可否直接挂载根目录啊

### DISCUSSION THREAD

**Discussion by kugimiya530** [AlistGo/alist#2992] [General]
因为根目录没有资料夹ID....

有没有办法直接挂载根目录

我会挂载个别目录

但我想挂载根目录  不知道表达的明不明白..

**Comment by NovaShen0724** [comment]
下次发Q&A
就填root也就是默认别动就行了吧

**Comment by kugimiya530** [comment]
刚刚又试了下 真的可以了

我原本也有用过root可是显示没有挂载到根目录

可能当时设置错了  感谢

NovaShen ***@***.***> 於 2023年1月12日 週四 上午12:04寫道：
